orca-core-tck/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/PipelineExecutionRepositoryTck.groovy

```diff
@@ -596,8 +596,12 @@ abstract class PipelineExecutionRepositoryTck<T extends ExecutionRepository> ext
 
   def "parses the parent execution of a pipeline trigger"() {
     given:
+    def parent = pipeline()
```

Comment:
It makes me slightly nervous that an existing test is changing as part of this.  Does this mean the feature is enabled by default, or?  What happens if this test doesn't change?

Reply:
This is wrong. I got this change from the original implementation but my changes are behind the feature flag and everything should work as expected. I am deleting this update because it is not needed.